search-service/src/test/kotlin/com/egm/stellio/search/service/AttributeInstanceServiceTests.kt

```diff
@@ -588,15 +612,15 @@ class AttributeInstanceServiceTests : WithTimescaleContainer, WithKafkaContainer
 
         attributeInstanceService.addAttributeInstance(
             incomingTemporalEntityAttribute.id,
-            "https://uri.etsi.org/ngsi-ld/default-context/hasBee",
+            attributeMetadata,
             attributeValues
         )
 
         verify {
             attributeInstanceService["create"](
                 match<AttributeInstance> {
                     it.time.toString() == "2015-10-18T11:20:30.000001Z" &&
-                        it.value == "false" &&
+                        it.value == false.toString() &&
```

Comment:
better keep the previous version (you want to check it has been transformed the way you expect it to)